src/actions/index.js

```diff
@@ -63,9 +63,11 @@ export default config => {
     return dispatch({ type: 'remove', path, previous })
   }
 
-  let mutate = _.curry(async (path, value) => {
+  let mutate = _.curry(async (path, value, isForceUpdate = false) => {
```

Comment:
has to use the false as the default because of the curry function. 
I also debating use isForceUpdate isForceMutate. or shouldForceUpdate